akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala

```diff
@@ -72,16 +83,15 @@ private[akka] object FlightRecorderReader {
       raFile = new RandomAccessFile(flightRecorderFile.toFile, "rw")
       channel = raFile.getChannel
       reader = new FlightRecorderReader(channel)
-      println(reader.structure)
-
-      println("--- ALERT ENTRIES")
-      reader.structure.alertLog.logs.foreach(log ⇒ println(log.richEntries.mkString("\n")))
+      val alerts: Seq[FlightRecorderReader#Entry] = reader.structure.alertLog.logs.flatMap(_.richEntries)
```

Comment:
I haven't actually seen a case where there is data in more than one log, so I'm not sure how it works.